### ISSUE TITLE
[FR] Add browser-based slicer support

### ISSUE THREAD

**Issue by VinnyCordeiro** [meteyou/mainsail#294]
That's an idea that appeared on Voron Discord. There's an open source slicer already available called Kiri:Moto, which is hosted here: https://github.com/GridSpace/grid-apps

It is browser-based, but not cloud-based, so the processing is made locally. Also, Kiri:Moto is licensed with the MIT License, which is compatible with GPL, so on that front that shouldn't be a problem.

The support could be added as a module, in order to not bloat the main installation for those who do not want this feature.

**Comment by meteyou** [comment]
Thx for you FR. Looks interesting!

**Comment by PhilippMolitor** [comment]
I've seen that OrcaSlicer supports being built for the web, there is some docker image by LinuxServer... could be an interesting integration to consider.

**Comment by meteyou** [comment]
@PhilippMolitor pls post a link to this information

**Comment by PhilippMolitor** [comment]
https://github.com/linuxserver/docker-orcaslicer

**Comment by Xolverr** [comment]
Any updates on this FR? Looking to use Kiri:Moto embedded into my web interface as well.

**Comment by meteyou** [comment]
@Xolverr no, and right now, i think i will not implement a slicer in Mainsail. If you like Kiri:Moto, you can use it side-by-side and it would only "blow up" Mainsail with only a very very low number of users. furthermore, there is a very strong difference in preference for slicers and i don't think that such good slicers as Orcaslicer or Prusaslicer manage to get the performance into a browser.
